src/ip_address.h

```diff
@@ -38,12 +38,7 @@
 #include <afunix.h>
 typedef unsigned short int sa_family_t;
 
-#elif __APPLE__
-#include <sys/socket.h>
-#include <netinet/in.h>
-#include <sys/un.h>      /* definitions for UNIX domain sockets */
-
-#elif __linux__
+#elif defined(__linux__) || defined(__APPLE__) 
```

Comment:
please mark the corresponding `#endif` with comment "__windows__, __linux__, __APPLE__" to make it easier to locate

Reply:
could you please apply this comment to all `#ifdef`s affected by your change?

Reply:
Updated!